### PR TITLE
new feature for back, search and reply in PyBitmessage Kivy

### PR DESCRIPTION
**Pull request by surbhicis**
# Repository contributions to the PyBitmessage project

- You can get paid for merged commits if you register at [Tip4Commit](https://tip4commit.com/github/Bitmessage/PyBitmessage)

### Code

- Try to refer to github issue tracker or other permanent sources of discussion about the issue.
- It is clear from the diff *what* you have done, it may be less clear *why* you have done it so explain why this change is necessary rather than what it does

### Documentation

- If there has been a change to the code, there's a good possibility there should be a corresponding change to the documentation 
- If you can't run `fab build_docs` successfully, ask for someone to run it against your branch

### Tests

- If there has been a change to the code, there's a good possibility there should be a corresponding change to the tests
- If you can't run `fab tests` successfully, ask for someone to run it against your branch

## Translations

- For helping with translations, please use [Transifex](https://www.transifex.com/bitmessage-project/pybitmessage/).
- There is no need to submit pull requests for translations.
- For translating technical terms it is recommended to consult the [Microsoft Language Portal](https://www.microsoft.com/Language/en-US/Default.aspx).

### Gitiquette

- Make the pull request against the ["v0.6" branch](https://github.com/Bitmessage/PyBitmessage/tree/v0.6)
- PGP-sign the commits included in the pull request
- Use references to tickets, e.g. `addresses #123` or `fixes #234` in your commit messages
- Try to use a good editor that removes trailing whitespace, highlights potential python issues and uses unix line endings
- If for some reason you don't want to use github, you can submit the patch using Bitmessage to the "bitmessage" chan, or to one of the developers.

### CODE REVIEW

**Comment by omkar1117** [comment]
@surbhicis what is the feature for?

**Comment by surbhicis** [comment]
Its a search for messages searching and a back button when we are on message creation screen and want to go back, with exit button changes with Action bar.

File: src/bitmessagekivy/mpybit.py [hunk 1]
```diff
@@ -38,8 +40,23 @@ def build(self):
         main_widget = Builder.load_file(
             os.path.join(os.path.dirname(__file__), 'main.kv'))
         self.nav_drawer = Navigator()
+        Window.bind(on_keyboard=self._key_handler)
         return main_widget
 
+    def _key_handler(self, instance, key, *args):
+        """Escape key manages previous screen on back."""
+        if key is 27:
```
Comment (omkar1117) [inline src/bitmessagekivy/mpybit.py]: Why can't we maintain doc string here??

**Comment by omkar1117** [comment]
@surbhicis any update here?

**Comment by surbhicis** [comment]
addcontact
connectioncount
createmessage
fillcontactsave
fromaddress
inbox
myaddress
navigation1
navigation2
process
randomaddress
settings
successsavecontact
validatioaddcontact

**Comment by surbhicis** [comment]
navigation1

navigation2

inbox

createmessage

inbox

myaddress

validatioaddcontact

successsavecontact

settings

randomaddress

process

fromaddress

connectioncount

fillcontactsave

validatioaddcontact

successsavecontact

**Comment by surbhicis** [comment]
addcontact
connectioncount
createmessage
fillcontactsave
fromaddress
inbox
myaddress
navigation1
navigation2
process

**Comment by g1itch** [comment]
It looks like the sources were randomly corrupted and sprinkled with garbage...

**Comment by omkar1117** [comment]
@surbhicis please resolve the conflicts.

**Pull request closed without merging**